thoughtis/cata-blocks
# Issue#123/decode rest urls

### Related Issues
- Closes #123 

### What Was Accomplished
- Decode HTML from API URLs on the edit screen for the REST block when a URL gets added and when posts are retrieved.

### How It Was Tested
- [x] Locally on parent theme
- [x] Locally on child theme
- [x] Child theme develop

### How To Test
- On the local main branch, create a REST API block and add a url with query parameters using `&amp;` instead of `&`. The displayed posts should ignore any query parameters.
- Switch to this branch and update the page with the REST API block without making any changes. The posts displayed in the editor and on the frontend should update to honor the URLs query parameters. The URL listed in the editor will still contain `&amp;`
- Test adding another URL to the REST API block that contains `&amp;`. It should be replaced with `&`.
